### ISSUE TITLE
Game Data Source?

### ISSUE THREAD

**Issue by mstksg** [SticklyMan/egginc-epic-research-calc#3]
Hi!  Thank you for this useful project :)

I was wondering where you got your game data source from?  Is it all acquired by hand, or did you find a source somewhere to look at?  Thought that you gathered it buy hand initially but saw that you were able to instantly respond to the new update, which might have been hard if you didn't have all of the golden eggs available at the time of the update :O 

Any help would be greatly appreciated :D

**Comment by SticklyMan** [comment]
Apologies for the delay in my reply! Keep in mind that I'm working on the Android platform. Initially, I was getting data from spreadsheets and my own observed values as I played, but several updates have changed previous values, so I had to change my methods.

I have tried to dig through the game data to find raw values, but I could not find anything. I'm quite certain that the values are calculated in functions within the binary-compiled core game code (.so), and disassembling that is far beyond my level of expertise (ain't nobody got time for that).

There are tools that I will not name that can circumvent in-app-purchases. They are sketchy and I would not trust them on my own personal device, but, luckily I have an old phone that I use for development which can be easily wiped, and does not connect to any of my personal accounts. With this, I can easily reset the game data, "buy" all of the golden eggs I need, then double check all epic research prices. Again, that is basically cheating, and I do not condone it except for the purposes of researching and collecting game data.

**Comment by mstksg** [comment]
No problem at all!  I did suspect as much.  I also tried digging through the apk, but I came to the same conclusion that you did.  I was even looking into learning how to decompile the binary, but then I took a moment to reconsider my life priorities :)

Nevertheless I definitely appreciate the research you've already done, and I hope you don't mind if I borrow some of the data (attributed to you, of course) for a tool I'm making? :)  [Gathering the data](https://github.com/mstksg/eggvisor/blob/ad8cc52b3378aca0d43c161ad35bd7d4f4cb40f9/data/gamedata.yaml#L569-L714) is definitely half of the "fun".
